setup.json

```
@@ -87,8 +86,8 @@
             "spglib~=1.14"
         ],
         "notebook": [
-            "jupyter==1.0.0",
-            "notebook<6"
+            "jupyter~=1.0.0",
+            "notebook~=6.0.0"
```

```suggestion
            "jupyter~=1.0",
            "notebook~=6.0"
```
We are maintaining the policy that if we support a major version of >=1 of a dependency, we always specify `~=1.*` which says basically any minor version of that major series goes, unless we have a very special reason that a certain minor version cannot be used or trusted.

---

Thanks! Changed.